setup.cfg

```diff
@@ -80,17 +81,13 @@ test =
     smart-open >= 5.0.0
     pyarrow >= 4.0.1
     six >= 1.11.0
-
 dask =
-    dask[dataframe] >= 2021.10.0
-
+    dask[dataframe] >= 2021.10.0, <2022.8.0
```

Comment:
Restricting dask for now (to get unit tests to pass):
https://github.com/alteryx/woodwork/issues/1498